exporter/awsprometheusremotewriteexporter/go.mod

```diff
@@ -4,11 +4,12 @@ go 1.17
 
 require (
 	github.com/aws/aws-sdk-go v1.42.23
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter v0.41.0
 	github.com/stretchr/testify v1.7.0
-	go.opentelemetry.io/collector v0.41.1-0.20211210184707-4dcb3388a168
+	go.opentelemetry.io/collector v0.41.1-0.20211218195202-81ab024486f0
 )
 
+require github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter v0.0.0-00010101000000-000000000000
```

Comment:
No replace that's why this is hapening?

Reply:
Script I ran downgraded the collector (because of #6843) and caused the require to be removed. When they were added again (because of the replace) they no longer contained a version number. I've closed this PR in favour of a simpler one that didn't cause this problem.